solutions/remotemonitoring/single-vm/scripts/auth.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+
+AUTH_SERVER_URL=""
+RESOURCE_TYPE=""
+AUTH_TOKEN=""
+
+# Acquires auth token for authroizating against key vault.
+_acquire_token() {
+    __set_keyvault_auth_server
+
+    local _temp=$(curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d "client_id=$PCS_AAD_APPID&resource=https%3A%2F%2Fvault.azure.net&client_secret=$PCS_AAD_APPSECRET&grant_type=client_credentials" $AUTH_SERVER_URL/oauth2/token)
+    AUTH_TOKEN=$(__parse_json $_temp "access_token")
+}
+
+# Fetch key vault secret.
+_set_keyvault_secret() {
+
+    # Get a new token each time you access key vault.
+    _acquire_token
+
+    _keyvault_secret_bundle=$(curl -X PUT -H "Content-Type: application/json" -H "Authorization: Bearer $AUTH_TOKEN" -d "{\"value\": $1}"  -L https://$PCS_KEYVAULT_NAME.vault.azure.net/secrets/authEnabled/?api-version=7.0)
+    _keyvault_secret_bundle="'$_keyvault_secret_bundle'"
+
+    _keyvault_secret_bundle=$(curl -X PUT -H "Content-Type: application/json" -H "Authorization: Bearer $AUTH_TOKEN" -d "{\"value\": $2}"  -L https://$PCS_KEYVAULT_NAME.vault.azure.net/secrets/corsWhitelist/?api-version=7.0)
+    _keyvault_secret_bundle="'$_keyvault_secret_bundle'"
```

Comment:
_keyvault_secret_bundle have values for both keys auth and cors? I would have thought that value on line 23 will be overwritten here.

Reply:
Yup, In fact, the line _keyvault_secret_bundle="'$_keyvault_secret_bundle'" is not required as we are not returning anything.

Reply:
I have removed it.